src/PREfastXmlSarifConverter/PREfastXmlSarifConverter.cpp

```diff
@@ -407,7 +407,7 @@ extern "C"
 HRESULT __stdcall Convert(const std::deque<XmlDefect> defectList, BSTR bstrOutputFile, BSTR* pbstrSarifText)
 {
     SarifLog issueLog;
-    issueLog.SetVersion(L"1.0.0-beta.4");
+    issueLog.SetVersion(L"1.0.0-beta.5");
     issueLog.SetSchema(L"http://json.schemastore.org/sarif-1.0.0");
```

Comment:
this value should now be [http://json.schemastore.org/sarif-1.0.0-beta.5](http://json.schemastore.org/sarif-1.0.0-beta.5)